crates/ruma-common/src/events/room/create.rs

```diff
@@ -50,16 +56,58 @@ pub struct RoomCreateEventContent {
 }
 
 impl RoomCreateEventContent {
-    /// Creates a new `RoomCreateEventContent` with the given creator.
-    pub fn new(creator: OwnedUserId) -> Self {
+    /// Creates a new `RoomCreateEventContent` with the given creator, as required for room versions
+    /// 1 through 10.
+    pub fn new_v1(creator: OwnedUserId) -> Self {
+        #[allow(deprecated)]
         Self {
-            creator,
+            creator: Some(creator),
```

Comment:
Can the `#[allow(deprecated)]` be on the field instead of on the whole struct literal?

Reply:
No, for some reason it still complains that the field is deprecated.